lib/_tls_wrap.js

```diff
@@ -228,7 +228,7 @@ function initRead(tls, wrapped) {
  */
 
 function TLSSocket(socket, options) {
-  this._tlsOptions = options;
+  this._tlsOptions = options || { isServer: false };
```

Comment:
If you would like to make `isServer` `false` by default - let's do it explicitly!